pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.impactupgrade.integration</groupId>
   <artifactId>hubspot-java-client</artifactId>
-  <version>2.1.0-SNAPSHOT</version>
+  <version>2.2.0-SNAPSHOT</version>
```

Comment:
No need to bump up the version until we do a release. 2.1.0.Final isn't out yet

Reply:
Removed